### PR TITLE
Update `Help_centre/Store`

### PR DESCRIPTION
**Pull request by pishifat**

### CODE REVIEW

File: wiki/Help_centre/Store/en.md [hunk 1]
```diff
@@ -2,124 +2,48 @@
...
+If your order has arrived in your country, but hasn't been delivered yet, it may be held at your local post office. Contact them and mention your tracking number for additional info! This is sometimes required for international purchases where customs or other taxes/duties are involved.
 
-### The LEDs on my osu! nono keyboard aren't working! {id=osu!keyboard-leds}
+Sometimes orders can get stuck in the postal system. If your order is stuck somewhere in the United States, it may be possible to track down through the [USPS](https://faq.usps.com/s/article/USPS-Tracking-The-Basics), usually by contacting the relevant location on your tracking info. If your order is stuck outside of the United States, contact your country's postal service for help.
 
-**This can be due to a few reasons - corrosion between the LEDs and the mainboard, or in certain circumstances with earlier models, faulty LEDs.**
+Feel free to contact [merch@ppy.sh](mailto:merch@ppy.sh) if you're still confused, but note that after a product leaves our hands, we knows as much about the status of the delivery as you do.
```
Comment (Walavouchey) [inline wiki/Help_centre/Store/en.md:31]:
```suggestion
Feel free to contact [merch@ppy.sh](mailto:merch@ppy.sh) if you're still confused, but note that after a product leaves our hands, we know as much about the status of the delivery as you do.
```

File: wiki/Help_centre/Store/en.md [hunk 2]
```diff
@@ -2,124 +2,48 @@
...
 
-*Main page: [Help centre](/wiki/Help_centre)*\
-*Note: These products are no longer available for purchase from the osu!store.*
+*Main page: [Help centre](/wiki/Help_centre)*
 
-Having trouble with a product from the osu!store? Check if there's an available solution to your problem!
+This article only applies to products purchased from the [osu!store](https://osu.ppy.sh/store/listing).
 
-## osu!keyboard {id=osu!keyboard}
+## I need to change my address!
```
Comment (Walavouchey) [inline wiki/Help_centre/Store/en.md:17]: i recommend adding new custom identifiers, such that section links always work even for people with different website language settings

**Review by Walavouchey — CHANGES_REQUESTED** [review]

File: wiki/Help_centre/id.md [hunk 3]
```diff
@@ -27,7 +27,7 @@ Harap tentukan jenis permasalahan yang paling menggambarkan masalah yang kamu ha
 | [Beatmapping dan Editor](/wiki/Help_centre/Beatmapping) | Pengelolaan beatmap, kepemilikan beatmap, sistem kuota beatmap (*beatmap slots*) |
 | [Klien](/wiki/Help_centre/Client) | *Bug* dan *crash*, permainan, koneksi, performa |
 | [Instalasi dan pendaftaran](/wiki/Help_centre/Installation_and_registration) | Pengunduhan, pembuatan akun |
-| [osu!store](/wiki/Help_centre/Store) | osu!go, osu!keyboard, osu!tablet, berbagai piranti dan cinderamata resmi lainnya |
+| [osu!store](/wiki/Help_centre/Store) | Berbagai piranti dan cinderamata resmi |
```
Comment (Nivalyx) [inline wiki/Help_centre/id.md:30]:
```suggestion
| [osu!store](/wiki/Help_centre/Store) | Cendera mata |
```

**Review by Walavouchey — DISMISSED** [review]

File: wiki/Help_centre/Store/en.md [hunk 4]
```diff
@@ -2,124 +2,48 @@
...
-### I've reformatted the osu!go device and it now shows up in Disk Management {id=osu!go-clean}
-
-**You're almost done!**
-
-Unzip the archive we asked you to download earlier (osu!go.zip) into the osu!go drive and you're done!
-
-### None of this worked! {id=osu!go-unknown-cause}
-
-If you encounter any issues with this process, please send an email to [store@ppy.sh](mailto:store@ppy.sh) and we'd be happy to help you out.
+That sucks. Contact [merch@ppy.sh](mailto:merch@ppy.sh) and we'll try to figure it out.
```
Comment (cl8n) [inline wiki/Help_centre/Store/en.md:49]: "that sucks" feels kind of out of place but idk, I can tell you wanted this page to feel more casual in tone than the average customer support page so maybe it's fine

**Review by cl8n — DISMISSED** [review]

File: wiki/Help_centre/Store/en.md [hunk 5]
```diff
@@ -2,124 +2,48 @@
...
 
-### My osu!tablet has stopped working or doesn't work at all! {id=osu-tablet-not-working}
+If your product has a problem of any sort, contact us at [merch@ppy.sh](mailto:merch@ppy.sh) with your order number and an explanation of the problem.
 
-**This can be tricky to troubleshoot since the osu!tablet is a two-unit solution (the tablet and the pen).**
+## None of this applies to what I need help with! {id=none}
 
-Given that the osu!tablet is a two-unit solution (e.g. tablet and pen), it can be difficult to tell which unit is experiencing issues when things go wrong.
-
-To find out, perform the following steps:
```
Comment (peppy) [inline wiki/Help_centre/Store/en.md:54]: Kinda sad to lose this historical context 😢

Could we perhaps make an archive / similar section at the bottom of the page? Or is it just noise at this point?

**Review by peppy — COMMENTED** [review]
Seems fine otherwise.

File: wiki/Help_centre/Store/en.md (hunk 5 above)
Comment (pishifat) [reply wiki/Help_centre/Store/en.md:54]: i re-added previous pages under `Help_centre/Store_archive`, i dont think it'd be good to crowd this page with stuff that's not really relevant anymore

**Comment by DanielPower** [comment]
Thank you for this, I really appreciate it!

File: wiki/Help_centre/Store/Store_archive/img/raw-status-osu-go.png
Comment (Joehuu) [inline wiki/Help_centre/Store/Store_archive/img/raw-status-osu-go.png:1]: Image from the old location was not deleted.

File: wiki/Help_centre/Store/en.md [hunk 6]
```diff
@@ -2,124 +2,49 @@
...
+  - keycaps
+  - deskmat
+  - deskmats
 ---
 
 # osu!store and merchandise
 
 *Main page: [Help centre](/wiki/Help_centre)*\
-*Note: These products are no longer available for purchase from the osu!store.*
+*See [store archive](/wiki/Help_centre/Store/Store_archive) for older product troubleshooting*
```
Comment (Joehuu) [inline wiki/Help_centre/Store/en.md:14]:
```suggestion
*For older product troubleshooting, see: [Store archive](/wiki/Help_centre/Store/Store_archive)*
```

File: wiki/Help_centre/Store/Store_archive/en.md [hunk 7]
```diff
@@ -0,0 +1,125 @@
...
+  - product
+  - keyboard
+  - tablet
+  - area
+  - device
+  - setup
+  - pendrive
+---
+
+# osu!store and merchandise
```
Comment (Joehuu) [inline wiki/Help_centre/Store/Store_archive/en.md:13]: Needs to be a different title, or wiki search will show 2 `osu!store and merchandise`:

Maybe this:
```suggestion
# Store archive
```

File: wiki/Help_centre/Store/Store_archive/en.md [hunk 8]
```diff
@@ -0,0 +1,125 @@
...
+
+*Main page: [Help centre](/wiki/Help_centre)*\
```
Comment (Joehuu) [inline wiki/Help_centre/Store/Store_archive/en.md:15]:
```suggestion
*Main page: [osu!store and merchandise](/wiki/Help_centre/Store)*\
```

**Review by Joehuu — APPROVED** [review]

**Pull request merged**